### PR TITLE
Project Machine created

### PR DESCRIPTION
**Pull request by fyarkiy**

### CODE REVIEW

File: src/main/java/core/basesyntax/Excavator.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    public void doWork() {
+        System.out.println("Excavator start working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stop working");
+    }
+}
+
```
Comment (boroda4436) [inline src/main/java/core/basesyntax/Excavator.java:18]: empty lines are redundant

File: src/main/java/core/basesyntax/Machine.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    public Machine(String plateNumber, int age) {
+        this.plateNumber = plateNumber;
+        this.age = age;
+    }
+
+    public abstract void doWork();
+
+    public abstract void stopWork();
+}
+
```
Comment (boroda4436) [inline src/main/java/core/basesyntax/Machine.java:16]: an extra empty line is redundant

File: src/main/java/core/basesyntax/MainApp.java [hunk 3]
```diff
@@ -12,5 +12,19 @@
...
+        Machine excavator = new Excavator("AA 4321AB",17);
+        track.doWork();
+        bulldozer.doWork();
+        excavator.doWork();
+        track.stopWork();
+        bulldozer.stopWork();
+        excavator.stopWork();
+    }
 }
+
```
Comment (boroda4436) [inline src/main/java/core/basesyntax/MainApp.java:28]: an extra empty lines is redundant

File: src/main/java/core/basesyntax/Excavator.java (hunk 1 above)
Comment (fyarkiy) [reply src/main/java/core/basesyntax/Excavator.java]: ok

File: src/main/java/core/basesyntax/MainApp.java (hunk 3 above)
Comment (fyarkiy) [reply src/main/java/core/basesyntax/MainApp.java]: ok

File: src/main/java/core/basesyntax/Machine.java (hunk 2 above)
Comment (fyarkiy) [reply src/main/java/core/basesyntax/Machine.java]: ok

**Pull request closed without merging**